File: preliminary_analysis/baseline_py_results/py_source/question-45-pass-0.py
# Sum of series formed by difference between product and sum of N natural numbers. 
def seriesSum(n): 
    return (n * (n + 1) * (2 * n + 1)) / 6 
  
# Driver code 
n = 10
print("Sum of series formed by difference between product and sum of N natural numbers is: ", 
      seriesSum(n)) 

# Output:
# Sum of series formed by difference between product and sum of N natural numbers is:  220

# Time Complexity: O(1)
# Space Complexity: O(1)

File: preliminary_analysis/baseline_cpp_results/cpp_source/question-45-pass-0.cpp
// Sum of series formed by difference between product and sum of N natural numbers. 
 #include <bits/stdc++.h> 
 using namespace std ;
 int seriesSum ( int n ) {
 
    int sum = 0 ;
    for ( int i = 1 ; i <= n ; i++ ) {
        sum += ( i * ( i + 1 ) ) ;
    }
    return sum ;
 }
 
 int productSum ( int n ) {
 
    int sum = 0 ;
    for ( int i = 1 ; i <= n ; i++ ) {
        sum += ( i * ( n + 1 - i ) ) ;
    }
    return sum ;
 }
 
 int main ( ) {
 
    int n ;
    cin >> n ;
    cout << seriesSum ( n ) - productSum ( n ) << endl ;
    return 0 ;
 }
